script.js

```diff
@@ -1,4 +1,82 @@
+var gamemode = 1;
+var player1Dice = [];
+var player2Dice = [];
+var player1Wc = "";
+var player2Wc = "";
+
+var diceRoller = function () {
+  var roll = Math.random() * 6;
+  var roundDown = Math.floor(roll) + 1;
+  return roundDown;
+};
+var newDice = function () {
+  var newDices = [diceRoller(), diceRoller()];
+  if (gamemode == 1) {
+    player1Dice = newDices;
+  } else if (gamemode == 3) {
+    player2Dice = newDices;
+  }
+  console.log(newDices);
+  return newDices;
+};
+
+var concatenateNumbers = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
+
+var playerChoice = function (input) {
+  if (gamemode == 2) {
+    if (input == `a`) {
+      player1Num = concatenateNumbers(player1Dice[0], player1Dice[1]);
+    } else if (input == `b`) {
+      player1Num = concatenateNumbers(player1Dice[1], player1Dice[0]);
+    }
+    player1Wc = player1Num;
+    return player1Num;
+  }
+};
+var playerChoice2 = function (input) {
```

Comment:
technically you can merge this with the previous function, as the contents are somewhat the same, and you can just add a small if statement to see which player is assigned the number